charts/camunda-platform/README.md

```diff
@@ -241,6 +241,7 @@ Check out the default [values.yaml](values.yaml) file, which contains the same c
 | | `ingress.tls` | Configuration for [TLS on the ingress resource](https://kubernetes.io/docs/concepts/services-networking/ingress/#tls) | |
 | | `ingress.tls.enabled` | If true, then TLS is configured on the ingress resource. If enabled the Ingress.host needs to be defined. | `false` |
 | | `ingress.tls.secretName` | Defines the secret name which contains the TLS private key and certificate | `""` |
+| | `elasticsearch.enabled` | Enable Elasticsearch deployment as part of the Camunda Platform Cluster | `true` |
```

Comment:
I moved this here to be alongside the `elasticsearch` configs. Not sure why to was added to the end of the table